manifests/charts/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -149,6 +149,10 @@ spec:
             value: "{{ $val }}"
           {{- end }}
           {{- end }}
+{{- if semverCompare "<1.19" .Capabilities.KubeVersion.GitVersion }}
+          - name: ENABLE_LEGACY_FSGROUP_INJECTION
```

Comment:
does a check for JWT Policy values need to be added too?

Reply:
It doesn't _need_ to be false -- it just _can_ be. IMO its not needed; this is strictly an improvement for Helm. While we could add the jwt check in helm as well, the jwt change is from Kubernetes 1.12 which is ~forever ago, so I am not sure its worth the effort

Reply:
okay thanks for the clarification